### ISSUE TITLE
index.html fails Accessibility Test

### ISSUE THREAD

**Issue by Yeab243** [Yeab243/github-issue-templates#5]
**Describe the bug**
The website does not comply with the United State's ADA & Section 508 accessibility legislation

**To Reproduce**
1. Go to 'https://www.accessibilitychecker.org/'
2. Paste in 'https://yeab243.github.io/github-issue-templates/'
3. Click Check Website
4. View error

**Expected behavior**
Accessibility Test should not show errors on page

**Screenshots**
Click on the following link to view the current Accessibility Test errors:

**Desktop (please complete the following information):**
 - N/A

**Smartphone (please complete the following information):**
 - N/A

**Additional context**
Landmark regions defined for the web page are not unique. The same name is provided for multiple landmarks.
